include/cron.php

```diff
@@ -1,8 +1,8 @@
 <?php
 use \Friendica\Core\Config;
 
-require_once("include/photos.php");
-require_once("include/user.php");
+require_once('include/photos.php');
```

Comment:
This is a good question. It should be unified across a single file. I'm not sure what the PSR-2 is telling about this. This is more a task for @Hypolite to answer.